RecoLocalFastTime/FTLRecProducers/plugins/MTDRecHitProducer.cc

```diff
@@ -83,7 +83,7 @@ void MTDRecHitProducer::produce(edm::Event& evt, const edm::EventSetup& es) {
     uint32_t flags = FTLRecHit::kGood;
     auto rechit = barrel_->makeRecHit(uhit, flags);
     if (flags == FTLRecHit::kGood)
-      barrelRechits->push_back(std::move(rechit));
```

Comment:
Getting rid of the temporary variable and calling makeRecHit in the push_back would seem to be the better fix.

Reply:
I think this won't work because `makeRecHit()` modifies the `flags` variable as a side effect, and then `flags` is used in the condition.